docs/components/ProviderSupport.vue

```diff
@@ -198,7 +198,7 @@ export default {
         {
           name: "Gemini",
           text: Supported,
-          structured: Planned,
+          structured: Supported,
```

Comment:
It's adapted support as it's not using model config at the moment. can you please add `Adapted` for now ?